app/src/main/java/com/elmeradrianv/shesafe/MainActivity.java

```diff
@@ -1,14 +1,63 @@
 package com.elmeradrianv.shesafe;
 
 import android.os.Bundle;
+import android.view.MenuItem;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+
+import com.elmeradrianv.shesafe.fragments.MapFragment;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
+import java.util.concurrent.atomic.AtomicReference;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
+
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        BottomNavigationView bottomNavigationView = findViewById(R.id.bottom_navigation);
+        final FragmentManager fragmentManager = getSupportFragmentManager();
+        // define your fragments here
+        final Fragment mapFragment = new MapFragment();
+//        final Fragment fragment2 = new SecondFragment();
```

Comment:
Commented out code

Reply:
This the guide for my next code. It's more simple to me work with phantom code to follow the same pattern of work